content/md/permissions-settings/access-rights-on-imports-exports_fr.md

```diff
@@ -0,0 +1,43 @@
+---
+id: access-rights-on-imports-exports
+themes: permissions-settings
+title: Définir les droits sur les **imports et exports**
+popular: false
+ee-only: true
+related: what-is-a-user-group, what-is-a-user, build-your-user-groups
+priority: low
+---
+
+# Introduction
+
+Vous pouvez facilement limiter l’édition ou l’exécution de profils d'[import](imports.html) et d'[export](exports.html) en fonction des [groupes d’utilisateurs](what-is-a-user-group.html) définis dans votre PIM.
+
+## Droits disponibles
+
+Il existe deux droits que vous pouvez accorder sur chaque profil d'import / export :
```

Comment:
```suggestion
Il existe deux droits que vous pouvez accorder aux utilisateurs sur chaque profil d'import / export :
```
Would be clearer, what do you think?

Reply:
In that case, I think it's better to say "aux groupes d'utilisateurs" because the rights are set to the user groups and not to the users.

Is it ok for you  ?

Reply:
Yes, you're totally right. Would be much clearer. I'll change my suggestion so that you can directly merge it in the PR using the "Commit suggestion button".